ceph.spec.in

```diff
@@ -563,6 +563,7 @@ Group:          System/Filesystems
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
 Requires:       python%{python3_pkgversion}-numpy
 Requires:       python3-scipy
+Requires:       python%{python3_pkgversion}-scikit-learn
```

Comment:
one last nit, could you move this line before `python3-scipy`, so they are ordered alphabetically.

Reply:
ah missed it, updated!